test/models/scholarship_application_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ScholarshipApplicationTest < ActiveSupport::TestCase
```

Comment:
There's one other empty test stub, service_test.rb, I feel like I should remove that one if I remove these two